rules/internal/testutils/parse.go

```diff
@@ -80,3 +102,69 @@ func ParseProto3Tmpl(t *testing.T, src string, data interface{}) *desc.FileDescr
 	// Parse the proto as a string.
 	return ParseProto3String(t, protoBytes.String())
 }
+
+// decompile reads compiled Go protos and returns a map of strings
+// corresponding to the original protos.
+//
+// This is used to handle google.api common protos, which some tests need
+// to use.
+func decompile(filenames ...string) map[string]string {
+	var fdps []*dpb.FileDescriptorProto
+	for _, filename := range filenames {
+		// Compiled golang protos include a compressed descriptor, which must be
+		// uncompressed before it can be populated into a FileDescriptorProto.
+		compressedDesc := proto.FileDescriptor(filename)
+		r, err := gzip.NewReader(bytes.NewReader(compressedDesc))
+		if err != nil {
+			panic(fmt.Errorf("Corrupted compressed file %q: %v", filename, err))
+		}
+		descBytes, _ := ioutil.ReadAll(r) // Error case is unreachable; inputs are trusted.
+
+		// Unmarshal the byte string into a FileDescriptorProto.
+		fdp := &dpb.FileDescriptorProto{}
+		proto.Unmarshal(descBytes, fdp)
```

Comment:
Check the error, e.g., `if err := proto.Unmarshal; err != nil { ... }`

Reply:
Done.